Dashboard/Views/ScrutineeringWindow.axaml

```diff
@@ -0,0 +1,129 @@
+             to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
+        <vm:ScrutineeringViewModel />
+    </Design.DataContext>
+    
+    <UserControl.Resources>
+        <!-- This allows the conditional rendering of the "DV Data" header -->
+        <utils:MeasurmentVisibility x:Key="MeasurmentVisibility" />
+    </UserControl.Resources>
+
+    <Panel>
+        <Grid>
+            <Grid.RowDefinitions>
+                <!-- Top 10% of the screen -->
+                <RowDefinition Height="0.1*" />
+                <!-- Auto sizes to the height of the content -->
+                <RowDefinition Height="Auto" />
+                <!-- Middle 70% of the screen (the carousel) -->
+                <RowDefinition Height="0.7*" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            <Grid.ColumnDefinitions>
+                <!-- 20% left margin -->
+                <ColumnDefinition Width="0.2*" />
+                <!-- Carousel takes 60% of horizontal space -->
+                <ColumnDefinition Width="0.6*" />
+                <!-- 20% right margin -->
+                <ColumnDefinition Width="0.2*" />
+            </Grid.ColumnDefinitions>
+            <StackPanel Grid.Row="0" Grid.Column="1" ZIndex="1" Margin="10 5">
+                <TextBlock TextAlignment="Left" Margin="0 30 0  0" Text="Autonomous Vehicle Inspection"
+                           FontSize="30"
+                           FontWeight="Bold"
+                           Foreground="White" />
+                <TextBlock TextAlignment="Left" Text="Functional Autonomous System Test (RES)"
+                           FontSize="25"
+                           FontWeight="Bold"
+                           Foreground="White" />
+            </StackPanel>
+            <Border Background="#3f3f3f"
+                    BorderBrush="Black"
+                    BorderThickness="4"
+                    CornerRadius="1"
+                    Padding="10" Margin="10"
+                    Grid.Row="1"
+                    Grid.Column="1">
+                <TextBlock FontWeight="Bold" TextAlignment="Center" Foreground="White"
+                           Text="{Binding YamlData.Top}" FontSize="18" />
+            </Border>
+            <Border Background="LightGray"
+                    BorderBrush="#2c2c2c"
+                    BorderThickness="4"
+                    CornerRadius="1"
+                    Padding="10" Margin="10"
+                    Grid.Row="2"
+                    Grid.Column="1">
+                <Carousel x:Name="slides" ItemsSource="{Binding YamlData.Steps}" HorizontalAlignment="Center"
+                          VerticalAlignment="Center" Foreground="#14080e" FontSize="18">
+                    <Carousel.PageTransition>
+                        <CompositePageTransition>
+                            <PageSlide Duration="0:00:01" Orientation="Horizontal" />
```

Comment:
When this transition happens, it moves the content on the existing page up / down depending on the next step

Reply:
Its not the transition that does this, its the vertical alignment as depending on the amount of content in the step it will adjust what is considered the "center"